.ci/docker/ci_commit_pins/triton-rocm.txt

```diff
@@ -1 +1 @@
-03a44c5c37c9cc9efcd4cfa83bbc11f5be480cf3
+380bfd61b65760885d0a5beaf6fa9c62690eff06
```

Comment:
Not sure from where this commit is picked up.
Checking this branch - https://github.com/ROCm/triton/commits/release/pytorch_2.1/
I see 03a44c5c37c9cc9efcd4cfa83bbc11f5be480cf3.

So I believe final commit will be after https://github.com/ROCm/triton/pull/540 is merged

Reply:
Yes, I updated the PR to pick up the commit after https://github.com/ROCm/triton/pull/540 merge